accessibility-checker-engine/help-v4/en-US/aria_attribute_redundant.html

```diff
@@ -74,13 +74,13 @@ <h3 id="ruleMessage"></h3>
 
 * [IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
 * [ARIA 1.2 specification - Conflicts with Host Language Semantics](https://www.w3.org/TR/wai-aria-1.2/#host_general_conflict)
-* [ARIA in HTML - Requirements for use of ARIA attributes in place of equivalent HTML attributes](https://www.w3.org/TR/html-aria/#docconformance-attr)
+* [ARIA in HTML - Requirements ARIA attributes in place of equivalent HTML attributes](https://www.w3.org/TR/html-aria/#docconformance-attr)
```

Comment:
The original was understandable. The correction doesn't make sense. "Requirements for" maybe?